spec/features/staff/users_can_view_reports_spec.rb

```diff
@@ -112,6 +112,35 @@
         expect(page).to have_content report.description
       end
 
+      scenario "the report shows the total forecasted and actual spend and the variance" do
+        quarter_one_2019 = Date.parse("2019-4-1")
+        quarter_two_2019 = Date.parse("2019-7-1")
+
+        activity = create(:project_activity, organisation: delivery_partner_user.organisation)
+
+        report = create(:report, :active, organisation: delivery_partner_user.organisation, fund: activity.associated_fund, financial_quarter: 1, financial_year: 2019, created_at: quarter_one_2019)
+        report_presenter = ReportPresenter.new(report)
+
+        _forecasted_value = create(:planned_disbursement, parent_activity: activity, period_start_date: quarter_one_2019, value: 1000)
```

Comment:
Should the planned disbursement also have `report: report` like the transaction below?

Reply:
No, I don't believe so. The planned disbursement for this financial quarter would not have been added in this report, rather in a previous one so I've just let the factory sort that.